### PR TITLE
Restructuring and rewording composite databases

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-629.surge.sh

File: modules/ROOT/pages/composite-databases/index.adoc [hunk 1]
```diff
@@ -1,11 +1,88 @@
...
-* xref:composite-databases/queries.adoc[Queries]
-* xref:composite-databases/considerations.adoc[Further considerations]
-* xref:composite-databases/sharding-with-copy.adoc[Sharding data with the `copy` command]
+[[composite-databases-introduction]]
+== Introduction
 
+A Composite database is a special type of database introduced in Neo4j 5.
+It supersedes the implementation of the previous Fabric in Neo4j 4.
 
+Fabric as a concept describes how different parts of a domain (graph data and computing resources), which can be in the same or multiple Neo4j DBMSs, can work as a unified system with a single access point.
```
Comment (tobias-johansson) [inline modules/ROOT/pages/composite-databases/index.adoc:12]: This sentence is complicated - can it be simplified?

File: modules/ROOT/pages/composite-databases/index.adoc [hunk 2]
```diff
@@ -1,11 +1,88 @@
...
+Composite databases are the means to access this partitioned data or graphs with a single Cypher query.
+
+Composite databases *do not store data* independently.
+They use _database aliases_, which can be local, remote, or sharded.
```
Comment (tobias-johansson) [inline modules/ROOT/pages/composite-databases/index.adoc:16]: What is "sharded" supposed to mean here? There are only two kinds of aliases atm

File: modules/ROOT/pages/composite-databases/index.adoc [hunk 3]
```diff
@@ -1,11 +1,88 @@
...
+| Transactions
+2+| Only transactions with queries that read from multiple graphs, or read from multiple graphs and write to a single graph, are allowed.
+
+| Neo4j embedded
+2+| Not available when Neo4j is used as an embedded database in Java applications. It can be used only in a typical client/server mode when users connect to a Neo4j DBMS from their client application or tool via Bolt or HTTP protocol.
+|===
+
+== Concepts
+
+The main concepts that one needs to understand when working with data using Composite databases are:
```
Comment (tobias-johansson) [inline modules/ROOT/pages/composite-databases/index.adoc:63]:
```suggestion
The main concepts that are relevant to understand when working with composite databases are:
```

File: modules/ROOT/pages/composite-databases/index.adoc [hunk 4]
```diff
@@ -1,11 +1,88 @@
...
 
-* xref:composite-databases/introduction.adoc[Introduction]
-* xref:composite-databases/queries.adoc[Queries]
-* xref:composite-databases/considerations.adoc[Further considerations]
-* xref:composite-databases/sharding-with-copy.adoc[Sharding data with the `copy` command]
+[[composite-databases-introduction]]
+== Introduction
 
+A Composite database is a special type of database introduced in Neo4j 5.
+It supersedes the implementation of the previous Fabric in Neo4j 4.
```
Comment (tobias-johansson) [inline modules/ROOT/pages/composite-databases/index.adoc:10]:
```suggestion
It supersedes the previous Fabric implementation in Neo4j 4.
```

File: modules/ROOT/pages/composite-databases/index.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/composite-databases/index.adoc]: Thank you so much, @tobias-johansson, for the review. Could you take a second look? I tried to explain it in a different way.

**Comment by renetapopova** [comment]
Approved by @izoratti  on Slack.

**Pull request merged**